services/QuillLMS/app/helpers/ethical_ai_helper.rb

```diff
@@ -0,0 +1,173 @@
+# frozen_string_literal: true
+
+module EthicalAIHelper
```

Comment:
This is purely take-or-leave, but you could just call this `EthicalAiHelper` without capitalizing the `i` in "AI", which will let you avoid the Zeitwerk override below.  I generally find that the  advantages of doing "normal" Rails stuff can help avoid future potential issues.

I doubt this is likely to come up with this particular code, so no need to change it if you don't want.